### PR TITLE
[RPC] Added two RPC methods for listing transactions

### PR DESCRIPTION
**Pull request by fenix2222**
Added implementation for

listtransactions
listsinceblock

RPC methods

### CODE REVIEW

File: src/Stratis.Bitcoin.Features.RPC/Models/TransactionInfoModel.cs [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+using Newtonsoft.Json;
+using Newtonsoft.Json.Linq;
+
+namespace Stratis.Bitcoin.Features.RPC.Models
+{
+    public class TransactionInfoModel
+    {
+        public TransactionInfoModel()
+        {
+
```
Comment (quantumagi) [inline src/Stratis.Bitcoin.Features.RPC/Models/TransactionInfoModel.cs:11]: Remove empty line.

**Review by quantumagi — APPROVED** [review]
LGTM

File: src/Stratis.Bitcoin.Features.RPC/Models/TransactionInfoModel.cs (hunk 1 above)
Comment (zeptin) [reply src/Stratis.Bitcoin.Features.RPC/Models/TransactionInfoModel.cs:11]:
```suggestion
```
Boot CI

**Comment by trevor-crypto** [comment]
Any update on when this will be merged and released?

**Pull request closed without merging**